feature/system/management/otg_tests/management_ha_test/management_ha_test.go

```diff
@@ -80,7 +83,14 @@ func TestManagementHA1(t *testing.T) {
 		true,
 		true,
 	)
-	if deviations.SetNoPeerGroup(dut) {
+	g := bs.DUTConf.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut)).GetOrCreateProtocol(cfgplugins.PTBGP, "BGP").GetOrCreateBgp().GetOrCreateGlobal()
```

Comment:
All vendors pass/accept/allow this configuration.
Reference from other test : https://github.com/openconfig/featureprofiles/blob/62e6b3981ff6e3a37edfb83a277084eec95c1480/feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go#L181
Let me know if you think otherwise.

Reply:
all of this is being done in internal/cfgplugins/bgp.go.
Global AFI/SAFI are being added here:
https://github.com/openconfig/featureprofiles/blob/0efc23f9f85558c880b2cf37bb8e435f31d40813/internal/cfgplugins/bgp.go#L413
But this does not have VPNv6 address family activated.

AS is being assigned here:
https://github.com/openconfig/featureprofiles/blob/0efc23f9f85558c880b2cf37bb8e435f31d40813/internal/cfgplugins/bgp.go#L420
https://github.com/openconfig/featureprofiles/blob/0efc23f9f85558c880b2cf37bb8e435f31d40813/internal/cfgplugins/bgp.go#L246

Router ID is being added here:
https://github.com/openconfig/featureprofiles/blob/0efc23f9f85558c880b2cf37bb8e435f31d40813/internal/cfgplugins/bgp.go#L222

Reply:
Is there a reason to enable BGP in the default vrf with regards to this test?

That's the point that I believe I'm missing.

If there isn't and it's not part of the test requirements, then it should be part of a deviation as it's "deviating" away from the test logic/expectation, regardless if the configuration block is accepted by vendors.

Reply:
[logutil_sent_pr3259.txt](https://github.com/user-attachments/files/16237565/logutil_sent_pr3259.txt)
Please see attached snippet of `-alsologtostderr -args -v=2` to make `logutil.go` print out the request.
As you can see for the `DEFAULT` vrf for global configs, as/router-id are missing while they are present for `mvrf1`

Reply:
cc @dplore 

The deviation / no deviation logic is typically based on the question, "what is the standard/openconfig-based approach of doing X".

It doesn't matter if all vendors support additional configuration, if this configuration is not compliant with the OC expectations. 

> But this does not have VPNv6 address family activated.

MP-BGP and VPNv6 are not a part of this test.

Reply:
The [README](https://github.com/openconfig/featureprofiles/blob/1bcd4d13e05f4e9b5230fe59aa5f5c17e3e795ab/feature/system/management/otg_tests/management_ha_test/README.md) currently does not require configuring BGP in the DEFAULT network-instance.  It looks like Cisco needs this added in order to also configure BGP on the 'mgmt' network-instance.  

So yes, this needs to be wrapped by a deviation (such as `bgp_in_default_ni_before_other_ni`), even though it is legal OC and accepted by other vendors.  

Deviations should be used for logic that is outside of the scope of the README and are intended to be removed.  

The README is ok as is on this topic.    There is no requirement in OC that BGP must be enabled in the DEFAULT network-instance before adding BGP into other network-instances.

Reply:
Sure @dplore @MarcCharlebois @LimeHat will add the Default NI AFIs under a deviation

Reply:
done

Reply:
Thanks, LGTM now